### ISSUE TITLE
Help text is partially hidden when use a combination of BooleanField and FieldPanel in page model

### ISSUE THREAD

**Issue by m1kola** [wagtail/wagtail#3522]
Help text is partially hidden when use a combination of `BooleanField` and `FieldPanel` in page model (see screenshots below).

To reproduce use the following snippet:

```python
class SomePage(Page):
    portrait_images_in_teaser = models.BooleanField(
        default=False, help_text='I am a long-long help text. I am so long so you would not '
                                 'be able to see me on your display. Surprise!'
    )

    settings_panels = Page.settings_panels + [
        FieldPanel('portrait_images_in_teaser'),
    ]
```

Desktop

<img width="636" alt="screen shot 2017-03-31 at 15 08 24" src="https://cloud.githubusercontent.com/assets/509198/24549864/0ceb945a-1624-11e7-9e6b-558bc605902e.png">

Tablet / mobile

<img width="664" alt="screen shot 2017-03-31 at 15 09 54" src="https://cloud.githubusercontent.com/assets/509198/24549889/237ace16-1624-11e7-9b13-d46b34d14df9.png">

**Comment by coredumperror** [comment]
Any movement on this? I just discovered this bug as I create a Boolean site setting with a long description, and I have yet to find a decent workaround.

**Comment by zerolab** [comment]
Are the changes in #3942 touching this? Worth giving it a test.

**Comment by coredumperror** [comment]
I did finally find a "workaround", though it won't work for everyone's needs. If you put your fields into a `MultiFieldPanel`, the help text doesn't hide itself, and just always displays in a single line under the `<input>` field. That works for my purposes, so I went with it.

**Comment by thibaudcolas** [comment]
I could still reproduce this, although in my testing there needs to be at least two fields on the panel. Full snippet:

```python
class SomePage(Page):
    portrait_images_in_teaser = models.BooleanField(
        default=False, help_text='1 I am a long-long help text. I am so long so you would not '
                                 'be able to see me on your display. Surprise! '
                                 '2 I am a long-long help text. I am so long so you would not '
                                 'be able to see me on your display. Surprise! '
                                 '3 I am a long-long help text. I am so long so you would not '
                                 'be able to see me on your display. Surprise! '
    )

    portrait_two = models.BooleanField(
        default=False, help_text='1 I am a long-long help text. I am so long so you would not '
                                 'be able to see me on your display. Surprise! '
                                 '2 I am a long-long help text. I am so long so you would not '
                                 'be able to see me on your display. Surprise! '
                                 '3 I am a long-long help text. I am so long so you would not '
                                 'be able to see me on your display. Surprise! '
    )

    settings_panels = Page.settings_panels + [
        FieldPanel('portrait_images_in_teaser'),
        FieldPanel('portrait_two'),
    ]
```

**Comment by thibaudcolas** [comment]
Also fix attempt over at #4855.

**Comment by thibaudcolas** [comment]
Fixed in #4855. This will be released in v2.4.
